prow/repoowners/repoowners.go

```diff
@@ -409,12 +400,35 @@ func (o *RepoOwners) walkFunc(path string, info os.FileInfo, err error) error {
 	return nil
 }
 
+// ParseOwners will open and unmarshal OWNERS into either a SimpleConfig or a FullConfig
+// Returns an error if the file cannot be read or unmarshalled
+func ParseOwners(path string) (SimpleConfig, FullConfig, error) {
+	simple := &SimpleConfig{}
```

Comment:
declare these as `var simple SimpleConfig` and take their address in the unmarshal step only?